src/Microsoft.AspNetCore.Razor.Evolution/Legacy/ITagHelperDescriptorResolver.cs

```diff
@@ -10,14 +10,6 @@ namespace Microsoft.AspNetCore.Razor.Evolution.Legacy
     /// </summary>
     internal interface ITagHelperDescriptorResolver
```

Comment:
The resolver just discovers TagHelpers, it doesn't need to understand directives.